README.md

```diff
@@ -1,6 +1,7 @@
 Git for Windows
 ===============
 
+[![Build status](https://git-for-windows.visualstudio.com/git/_apis/build/status/12?branch=master)](https://git-for-windows.visualstudio.com/git/_build/latest?definitionId=12&branch=master)]
```

Comment:
The stray `]` at the end of this line is not necessary for rendering the badge, and can be removed:

<img width="409" alt="screen shot 2018-07-06 at 9 59 19 am" src="https://user-images.githubusercontent.com/359239/42380163-5333c64e-8103-11e8-9e9d-759d782ef8be.png">

Reply:
D'oh!

Reply:
Fixed.